deployments/icesat2/config/staging.yaml

```diff
@@ -4,7 +4,7 @@ pangeo:
     proxy:
       https:
         hosts:
-        - staging.icesat2.pangeo.io
+        - icesat2.pangeo.io
```

Comment:
this change should be in .../config/prod.yaml.

Reply:
i think we're good to go now